robot/web/web.go

```diff
@@ -606,8 +606,13 @@ func (svc *webService) startStream(streamFunc func(opts *webstream.BackoffTuning
 
 func (svc *webService) startImageStream(ctx context.Context, source gostream.VideoSource, stream gostream.Stream) {
 	ctxWithJPEGHint := gostream.WithMIMETypeHint(ctx, rutils.WithLazyMIMEType(rutils.MimeTypeJPEG))
+	actualCam := rutils.UnwrapProxy(source).(camera.Camera)
```

Comment:
Are you sure that _all_ callers of `startImageStream` will pass in a `gostream.VideoSource` whose underlying type is a `camera.videoSource`? The function `camera.SourceFromCamera` requires that but I'm not sure where this method is or will be used.